tools/TestImpact/Test-CiGateModes.ps1

```
@@ -0,0 +1,112 @@
+        $env:PROMOTION_RESULT = $Promotion
+        $env:SONAR_RESULT = 'success'
+        $env:REUSED_VALIDATION = $Reuse
+        $env:GITHUB_STEP_SUMMARY = Join-Path $fixture "$Name-summary.md"
+
+        & $bash $script 2>&1 | Out-Null
+        if ($LASTEXITCODE -ne $ExpectedExit) {
+            [void] $failures.Add("$Name expected exit $ExpectedExit, got $LASTEXITCODE")
+        }
+    }
+
+    Invoke-GateCase reuse_success true success skipped skipped false 0
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Invoke-GateCase' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)